cmd/platform/test.go

```diff
@@ -46,6 +55,23 @@ func webClientTestsCmdF(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func serverForWebClientTestsCmdF(cmd *cobra.Command, args []string) error {
+	initDBCommandContextCobra(cmd)
+	utils.InitTranslations(utils.Cfg.LocalizationSettings)
+	api.InitRouter()
+	api.InitApi()
```

Comment:
Can you also spin up api4 here with `api4.InitApi(false)`? Make sure it's above `api.InitApi()`